Content.Server/_Goobstation/Changeling/ChangelingSystem.cs

```diff
@@ -693,7 +693,17 @@ private void OnStartup(EntityUid uid, ChangelingComponent comp, ref ComponentSta
     private void OnMobStateChange(EntityUid uid, ChangelingComponent comp, ref MobStateChangedEvent args)
     {
         if (args.NewMobState == MobState.Dead)
+        {
             RemoveAllChangelingEquipment(uid, comp);
+            // Automatically put the ling into stasis on death if there's enough biomass
+            if (!comp.IsInStasis && !HasComp<AbsorbedComponent>(uid) && comp.Biomass > 1)
+            {
+                comp.Chemicals = 0f;
+                comp.Biomass -= 1;
```

Comment:
pr seems fine overall but changelings already lose biomass waiting to be able to exit stasis, being forced to lose a minute on death is just a bit rougher for them on top of that

also now that i look at it, changelings don't get `IsInStasis` reset from other revives like defibs? goobcode torments me

Reply:
after looking a little harder i now realise that the action already cost 1 biomass anyway. so this is fine yay!!